<!doctype html><html lang=en-us><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge,chrome=1"><title>Shotgun Metagenome Analyses | Ikigai Informatics</title><meta name=viewport content="width=device-width,minimum-scale=1"><meta name=generator content="Hugo 0.75.1"><meta name=ROBOTS content="INDEX, FOLLOW"><link href=https://ikigaiinformatics.github.io/dist/css/app.4fc0b62e4b82c997bb0041217cd6b979.css rel=stylesheet><link rel="shortcut icon" href=https://ikigaiinformatics.github.io/logo.png type=image/x-icon><meta property="og:title" content="Shotgun Metagenome Analyses"><meta property="og:description" content="Deep Community Profiling via Shotgun Metagenome"><meta property="og:type" content="article"><meta property="og:url" content="https://ikigaiinformatics.github.io/analyses/metagenome/metagenome/"><meta itemprop=name content="Shotgun Metagenome Analyses"><meta itemprop=description content="Deep Community Profiling via Shotgun Metagenome"><meta itemprop=wordCount content="234"><meta itemprop=keywords content><meta name=twitter:card content="summary"><meta name=twitter:title content="Shotgun Metagenome Analyses"><meta name=twitter:description content="Deep Community Profiling via Shotgun Metagenome"></head><body class="ma0 avenir bg-near-white production"><header><div class=bg-dark-green><nav class="pv3 ph3 ph4-ns" role=navigation><div class="flex-l justify-between items-center center"><a href=https://ikigaiinformatics.github.io/ class="f3 fw2 hover-white no-underline white-90 dib">Ikigai Informatics</a><div class="flex-l items-center"><ul class="pl0 mr3 site-menu"><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=https://ikigaiinformatics.github.io/about/ title="About page">About</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=https://ikigaiinformatics.github.io/bioinformatics/ title="Bioinformatics page">Bioinformatics</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=https://ikigaiinformatics.github.io/it/ title="Computing page">Computing</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=https://ikigaiinformatics.github.io/contact/ title="Contact page">Contact</a></li></ul></div></div></nav></div></header><main class=pb7 role=main><div class="flex-l mt2 mw8 center"><article class="center cf pv5 ph3 ph4-ns mw7"><header><p class="f6 b helvetica tracked">ANALYSES</p><h1 class=f1>Shotgun Metagenome Analyses</h1></header><div class="nested-copy-line-height lh-copy f4 nested-links nested-img mid-gray"><p>We perform Shotgun Metagenome Studies to generate the following information about the prospective communities:</p><ol><li><p>Data QC and Contamination screen: Low quality sequences and bases are removed. Subsequently, if the samples are obtained from a host, we remove reads related to the host sequences.</p></li><li><p>Meta-Genome Assembly: Create contiguous sequence segments using the sequenced reads representing the communities. During this step, data from all the samples are pooled for the assemblies.</p></li><li><p>Binning the Contigs and Curation: Contigs from the above step are segregaed into Bins based on the contig sequence properties. Curation is done by consolidating multiple binning approaches and removing contaminant sequences from the Bins.</p></li><li><p>Blob Analyses: Colorful community visualization based on sequence properties (eg. GC content), nucleotide frequencies and bin abundances.</p></li><li><p>Bin Reassembly: Curated bins are reassembled using strict and permissive filters via constituent reads to create a high quality Metagenome Assembled Genomes (MAGs).</p></li><li><p>Taxonomic Assignments: Assign taxonomies to the curated/reassembled bins.</p></li><li><p>Annotation of the Bins: Predict functional and non-functional features of the MAGs. Genes, CRISPR arrays, tRNAs, rRNAs etc are predicted.</p></li><li><p>Quantification of Bins: Bins will be quantified to estimate the abundance of the MAGs.</p></li><li><p>Functional Enrichment Analyses: Identify enriched functional pathways in the community, these can be KEGG, COG, CAZy, GO features etc. Study how the community is functionally interacting.</p></li><li><p>Differential Abundance Analyses: Identify the significantly modulated community member(s) in the metagenome and try to assess their impacts.</p></li></ol><p><strong>Analyses pipeline at Ikigai</strong>
<link rel=stylesheet href=https://ikigaiinformatics.github.io/css/hugo-easy-gallery.css><div class=box><figure itemprop=associatedMedia itemscope itemtype=http://schema.org/ImageObject><div class=img><img itemprop=thumbnail src=https://ikigaiinformatics.github.io/images/Metagenome.svg></div><a href=https://ikigaiinformatics.github.io/images/Metagenome.svg itemprop=contentUrl></a></figure></div></p></div></article></div></main><footer class="bg-dark-green bottom-0 w-100 pa3" role=contentinfo><div class="flex justify-between"><a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href=https://ikigaiinformatics.github.io/>&copy; Ikigai Informatics 2020</a><div></div></div></footer><script src=https://ikigaiinformatics.github.io/dist/js/app.3fc0f988d21662902933.js></script></body></html>