<!doctype html><html lang=en-us><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge,chrome=1"><title>Exome Analyses | Ikigai Informatics</title><meta name=viewport content="width=device-width,minimum-scale=1"><meta name=generator content="Hugo 0.75.1"><meta name=ROBOTS content="NOINDEX, NOFOLLOW"><link href=/dist/css/app.4fc0b62e4b82c997bb0041217cd6b979.css rel=stylesheet><meta property="og:title" content="Exome Analyses"><meta property="og:description" content="We offer the following approach to analyse Exome data. The primary requisite information is the kit that was used to prepare the library and the associated BED file so as to identify the targeted regions.
  Alignment of the reference genome: Depending on the kit and the associated BED, we will be aligning the reads to the appropriate version: Hg19, Hg38, Grch37, Grch38.
  Variation calling: For Tumor-Normal matched samples, somatic variation calling will be done."><meta property="og:type" content="article"><meta property="og:url" content="https://ikigaiinformatics.github.io/analyses/exome/exome/"><meta itemprop=name content="Exome Analyses"><meta itemprop=description content="We offer the following approach to analyse Exome data. The primary requisite information is the kit that was used to prepare the library and the associated BED file so as to identify the targeted regions.
  Alignment of the reference genome: Depending on the kit and the associated BED, we will be aligning the reads to the appropriate version: Hg19, Hg38, Grch37, Grch38.
  Variation calling: For Tumor-Normal matched samples, somatic variation calling will be done."><meta itemprop=wordCount content="141"><meta itemprop=keywords content><meta name=twitter:card content="summary"><meta name=twitter:title content="Exome Analyses"><meta name=twitter:description content="We offer the following approach to analyse Exome data. The primary requisite information is the kit that was used to prepare the library and the associated BED file so as to identify the targeted regions.
  Alignment of the reference genome: Depending on the kit and the associated BED, we will be aligning the reads to the appropriate version: Hg19, Hg38, Grch37, Grch38.
  Variation calling: For Tumor-Normal matched samples, somatic variation calling will be done."></head><body class="ma0 avenir bg-near-white"><header><div class=bg-black><nav class="pv3 ph3 ph4-ns" role=navigation><div class="flex-l justify-between items-center center"><a href=https://ikigaiinformatics.github.io/ class="f3 fw2 hover-white no-underline white-90 dib">Ikigai Informatics</a><div class="flex-l items-center"><ul class="pl0 mr3 site-menu"><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=/about/ title="About page">About</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=/contact/ title="Contact page">Contact</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=/services/ title="Services page">Services</a></li></ul></div></div></nav></div></header><main class=pb7 role=main><div class="flex-l mt2 mw8 center"><article class="center cf pv5 ph3 ph4-ns mw7"><header><p class="f6 b helvetica tracked">ANALYSES</p><h1 class=f1>Exome Analyses</h1></header><div class="nested-copy-line-height lh-copy f4 nested-links nested-img mid-gray"><p>We offer the following approach to analyse Exome data. The primary requisite information is the kit that was used to prepare the library and the associated BED file so as to identify the targeted regions.</p><ol><li><p>Alignment of the reference genome: Depending on the kit and the associated BED, we will be aligning the reads to the appropriate version: Hg19, Hg38, Grch37, Grch38.</p></li><li><p>Variation calling: For Tumor-Normal matched samples, somatic variation calling will be done. Germline variation calling will be done for an unmatched sample.</p></li><li><p>Copy number identification: For unmatched samples, we will be using HMM based approach to identify if there are Copy Number Variations if any in the sample.</p></li><li><p>Annotation of Variation: Variations will be annotated against a plethora of databases like OMIM, COSMIC, dbSNP, Sift and LoFT to assess the impact of the variants observed.</p></li></ol><p><strong>Analyses pipeline at Ikigai</strong>
<link rel=stylesheet href=/css/hugo-easy-gallery.css><div class=box><figure itemprop=associatedMedia itemscope itemtype=http://schema.org/ImageObject><div class=img><img itemprop=thumbnail src=/images/Exome.svg></div><a href=/images/Exome.svg itemprop=contentUrl></a></figure></div></p></div></article></div></main><footer class="bg-black bottom-0 w-100 pa3" role=contentinfo><div class="flex justify-between"><a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href=https://ikigaiinformatics.github.io/>&copy; Ikigai Informatics 2020</a><div></div></div></footer><script src=/dist/js/app.3fc0f988d21662902933.js></script></body></html>