<!doctype html><html lang=en-us><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge,chrome=1"><title>Ikigai Informatics</title><meta name=viewport content="width=device-width,minimum-scale=1"><meta name=generator content="Hugo 0.75.1"><meta name=ROBOTS content="NOINDEX, NOFOLLOW"><link href=/dist/css/app.4fc0b62e4b82c997bb0041217cd6b979.css rel=stylesheet><link href=/analyses/index.xml rel=alternate type=application/rss+xml title="Ikigai Informatics"><link href=/analyses/index.xml rel=feed type=application/rss+xml title="Ikigai Informatics"><meta property="og:title" content="Analyses"><meta property="og:description" content="Inquest Immaculate, Bioinformatics, Survey, Conservation, Genomics, Metagenomics, Transcriptomics, Omics, miRNA, pacbio, hifi, clr, illumina, 10x, nanopore, ont, amplicon, 16s, ITS, hyper variable regions, genome, annotation, denovo, smallRNA, resequencing"><meta property="og:type" content="website"><meta property="og:url" content="https://ikigaiinformatics.github.io/analyses/"><meta itemprop=name content="Analyses"><meta itemprop=description content="Inquest Immaculate, Bioinformatics, Survey, Conservation, Genomics, Metagenomics, Transcriptomics, Omics, miRNA, pacbio, hifi, clr, illumina, 10x, nanopore, ont, amplicon, 16s, ITS, hyper variable regions, genome, annotation, denovo, smallRNA, resequencing"><meta name=twitter:card content="summary"><meta name=twitter:title content="Analyses"><meta name=twitter:description content="Inquest Immaculate, Bioinformatics, Survey, Conservation, Genomics, Metagenomics, Transcriptomics, Omics, miRNA, pacbio, hifi, clr, illumina, 10x, nanopore, ont, amplicon, 16s, ITS, hyper variable regions, genome, annotation, denovo, smallRNA, resequencing"></head><body class="ma0 avenir bg-near-white"><header><div class="pb3-m pb6-l bg-black"><nav class="pv3 ph3 ph4-ns" role=navigation><div class="flex-l justify-between items-center center"><a href=https://ikigaiinformatics.github.io/ class="f3 fw2 hover-white no-underline white-90 dib">Ikigai Informatics</a><div class="flex-l items-center"><ul class="pl0 mr3 site-menu"><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=/about/ title="About page">About</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=/contact/ title="Contact page">Contact</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=/services/ title="Services page">Services</a></li></ul></div></div></nav><div class="tc-l pv3 ph3 ph4-ns"><h1 class="f2 f-subheadline-l fw2 light-silver mb0 lh-title">Analyses</h1></div></div></header><main class=pb7 role=main><article class="pa3 pa4-ns nested-copy-line-height nested-img"><section class="cf ph3 ph5-l pv3 pv4-l f4 tc-l center measure-wide lh-copy mid-gray"></section><section class="flex-ns flex-wrap justify-around mt5"><div class="relative w-100 w-30-l mb4 bg-white"><div class="relative w-100 mb4 bg-white nested-copy-line-height"><div class="bg-white mb3 pa4 gray overflow-hidden"><span class="f6 db">Analyses</span><h1 class="f3 near-black"><a href=https://ikigaiinformatics.github.io/analyses/conservation_genomics/conservation_genomics/ class="link black dim">Conservation Genomics</a></h1><div class="nested-links f5 lh-copy nested-copy-line-height">At Ikigai we offer a special analyses pipeline for creating resources that might help saving flora and fauna from getting extinct.
It is due to our actions, directly or indirectly various factors have arisen that have led to many species getting wiped out. 1 2
Analyses flow at Ikigai:
QC of the datasets to remove low quality data.
Genome Survey analyses to identify various pecularities of the genome.</div></div></div></div><div class="relative w-100 w-30-l mb4 bg-white"><div class="relative w-100 mb4 bg-white nested-copy-line-height"><div class="bg-white mb3 pa4 gray overflow-hidden"><span class="f6 db">Analyses</span><h1 class="f3 near-black"><a href=https://ikigaiinformatics.github.io/analyses/rnaseq_denovo/rnaseq_denovo/ class="link black dim">Denovo RNASeq</a></h1><div class="nested-links f5 lh-copy nested-copy-line-height">We offer the following approach to analyse RNASeq data if the reference genome is not available at public resources like NCBI, ENSEMBL, JGI or others. This might be due to a variety of reasons including lack of a reference altogether amongst many.
Normalization of Reads: Reads will be normalized using a standalone approach to ease the computational load and to correct for coverage of ultra-high/ultra-low kmers to assemble these transcripts.</div></div></div></div><div class="relative w-100 w-30-l mb4 bg-white"><div class="relative w-100 mb4 bg-white nested-copy-line-height"><div class="bg-white mb3 pa4 gray overflow-hidden"><span class="f6 db">Analyses</span><h1 class="f3 near-black"><a href=https://ikigaiinformatics.github.io/analyses/eukaryote_assembly/eukaryote_assembly/ class="link black dim">Eukaryotic Genome Assembly</a></h1><div class="nested-links f5 lh-copy nested-copy-line-height">We perform Eukaryotic Genome Assemblies to generate the following information about the prospective organism.
Genome Survey: Determine metrics like Genome size, heterozygosity and repetitive content of the genome.
Genome Assembly: Create contiguous sequence segments using the sequenced reads representing the genome. Assembly will be validated using BUSCO, Transcript mapping (subject to availability) and Merqury to assess the genome completion.
Genome Masking: Identify low complexity, mobile elements and simple repeats to mine features.</div></div></div></div><div class="relative w-100 w-30-l mb4 bg-white"><div class="relative w-100 mb4 bg-white nested-copy-line-height"><div class="bg-white mb3 pa4 gray overflow-hidden"><span class="f6 db">Analyses</span><h1 class="f3 near-black"><a href=https://ikigaiinformatics.github.io/analyses/exome/exome/ class="link black dim">Exome Analyses</a></h1><div class="nested-links f5 lh-copy nested-copy-line-height">We offer the following approach to analyse Exome data. The primary requisite information is the kit that was used to prepare the library and the associated BED file so as to identify the targeted regions.
Alignment of the reference genome: Depending on the kit and the associated BED, we will be aligning the reads to the appropriate version: Hg19, Hg38, Grch37, Grch38.
Variation calling: For Tumor-Normal matched samples, somatic variation calling will be done.</div></div></div></div><div class="relative w-100 w-30-l mb4 bg-white"><div class="relative w-100 mb4 bg-white nested-copy-line-height"><div class="bg-white mb3 pa4 gray overflow-hidden"><span class="f6 db">Analyses</span><h1 class="f3 near-black"><a href=https://ikigaiinformatics.github.io/analyses/genome_survey/genome_survey/ class="link black dim">Genome Survey</a></h1><div class="nested-links f5 lh-copy nested-copy-line-height">Genome Survey analyses at Ikigai entails extraction of the following information from the datasets using a kmer based approach.
Since Genome Survey is done at 25-30X data (i.e 1Gb genome => 30Gb data), we can analyse the data in the below fashion.
Genome size: Ballpark the size of the genome and mark it against existing literature.
Heterozygosity of genome: Heterozygosity corelates with the presence of allelic variations, which might complicate genome assembly.</div></div></div></div><div class="relative w-100 w-30-l mb4 bg-white"><div class="relative w-100 mb4 bg-white nested-copy-line-height"><div class="bg-white mb3 pa4 gray overflow-hidden"><span class="f6 db">Analyses</span><h1 class="f3 near-black"><a href=https://ikigaiinformatics.github.io/analyses/prokaryote_assembly/prokaryote_assembly/ class="link black dim">Prokaryotic Genome Assembly</a></h1><div class="nested-links f5 lh-copy nested-copy-line-height">We perform Prokaryotic Genome Assemblies to generate the following information about the prospective organism.
Data QC and Contamination screen: Low quality sequences and bases are removed. Subsequently 10000 reads are picked randomly to screen and calculate the proportion of the organism of interest.
Genome Assembly: Create contiguous sequence segments using the sequenced reads representing the genome. Please let us know if Plasmids are also present in the data.</div></div></div></div><div class="relative w-100 w-30-l mb4 bg-white"><div class="relative w-100 mb4 bg-white nested-copy-line-height"><div class="bg-white mb3 pa4 gray overflow-hidden"><span class="f6 db">Analyses</span><h1 class="f3 near-black"><a href=https://ikigaiinformatics.github.io/analyses/dnaseq_reference/dnaseq_reference/ class="link black dim">Reference Aided DNASeq</a></h1><div class="nested-links f5 lh-copy nested-copy-line-height">We offer the following approach to analyse WGS data if the reference genome is readily available at public resources like NCBI, ENSEMBL, JGI or others.
Alignment of the reference genome: Reads will be aligned to the reference genome to check the mappability, coverage and depth of the organism sequenced.
Variation calling: For the samples, we will be performing variant calling to call SNVs, InDels and smaller complex variations like MNPs (&lt;50bp).</div></div></div></div><div class="relative w-100 w-30-l mb4 bg-white"><div class="relative w-100 mb4 bg-white nested-copy-line-height"><div class="bg-white mb3 pa4 gray overflow-hidden"><span class="f6 db">Analyses</span><h1 class="f3 near-black"><a href=https://ikigaiinformatics.github.io/analyses/rnaseq_reference/rnaseq_reference/ class="link black dim">Reference Aided RNASeq</a></h1><div class="nested-links f5 lh-copy nested-copy-line-height">We offer the following approach to analyse RNASeq data if the reference genome is readily available at public resources like NCBI, ENSEMBL, JGI or others. We prefer having the genome from ENSEMBL due to the downstream integration with biomart.
Alignment of Reads: Reads will be aligned with the genome using the splice-sites information if available depending on the organism (prokaryotes will generally not have).
Novel transcript assembly: Novel transcripts will be assembled using the help of the genome and screened for their coding potential.</div></div></div></div><div class="relative w-100 w-30-l mb4 bg-white"><div class="relative w-100 mb4 bg-white nested-copy-line-height"><div class="bg-white mb3 pa4 gray overflow-hidden"><span class="f6 db">Analyses</span><h1 class="f3 near-black"><a href=https://ikigaiinformatics.github.io/analyses/mirna/mirna/ class="link black dim">smallRNA Analyses</a></h1><div class="nested-links f5 lh-copy nested-copy-line-height">We offer the following approach to analyse RNASeq data if the reference genome is readily available at public resources like NCBI, ENSEMBL, JGI or others. We prefer having the genome from ENSEMBL due to the downstream integration with biomart.
QC and Read Trimming: Trim adapters and retain reads between 18-30bp.
Alignment to other ncRNAs: ncRNA catalogue from which the smallRNA sequences (precursors or mature) have been removed are used as a bait to remove reads originating from such sequences.</div></div></div></div></section></article></main><footer class="bg-black bottom-0 w-100 pa3" role=contentinfo><div class="flex justify-between"><a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href=https://ikigaiinformatics.github.io/>&copy; Ikigai Informatics 2020</a><div></div></div></footer><script src=/dist/js/app.3fc0f988d21662902933.js></script></body></html>