<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Denovo RNASeq | Ikigai Informatics</title>
    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="generator" content="Hugo 0.75.1" />
    
    
      <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
    

    
    
      <link href="/dist/css/app.4fc0b62e4b82c997bb0041217cd6b979.css" rel="stylesheet">
    

    

    
      

    

    
    
    <meta property="og:title" content="Denovo RNASeq" />
<meta property="og:description" content="We offer the following approach to analyse RNASeq data if the reference genome is not available at public resources like NCBI, ENSEMBL, JGI or others. This might be due to a variety of reasons including lack of a reference altogether amongst many.
  Normalization of Reads: Reads will be normalized using a standalone approach to ease the computational load and to correct for coverage of ultra-high/ultra-low kmers to assemble these transcripts." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://ikigaiinformatics.github.io/analyses/details/rnaseq_denovo/" />

<meta itemprop="name" content="Denovo RNASeq">
<meta itemprop="description" content="We offer the following approach to analyse RNASeq data if the reference genome is not available at public resources like NCBI, ENSEMBL, JGI or others. This might be due to a variety of reasons including lack of a reference altogether amongst many.
  Normalization of Reads: Reads will be normalized using a standalone approach to ease the computational load and to correct for coverage of ultra-high/ultra-low kmers to assemble these transcripts.">

<meta itemprop="wordCount" content="339">



<meta itemprop="keywords" content="" />
<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Denovo RNASeq"/>
<meta name="twitter:description" content="We offer the following approach to analyse RNASeq data if the reference genome is not available at public resources like NCBI, ENSEMBL, JGI or others. This might be due to a variety of reasons including lack of a reference altogether amongst many.
  Normalization of Reads: Reads will be normalized using a standalone approach to ease the computational load and to correct for coverage of ultra-high/ultra-low kmers to assemble these transcripts."/>

	
  </head>

  <body class="ma0 avenir bg-near-white">

    

  <header>
    <div class="bg-black">
      <nav class="pv3 ph3 ph4-ns" role="navigation">
  <div class="flex-l justify-between items-center center">
    <a href="https://ikigaiinformatics.github.io/" class="f3 fw2 hover-white no-underline white-90 dib">
      Ikigai Informatics
    </a>
    <div class="flex-l items-center">

      
      
      <ul class="pl0 mr3 site-menu">
        
            
              <li class="list f5 f4-ns fw4 dib pr3">
                <a class="hover-white no-underline white-90" href="/about/" title="About page">
                  About
                </a>
              </li>

            
        
            
              <li class="list f5 f4-ns fw4 dib pr3">
                <a class="hover-white no-underline white-90" href="/contact/" title="Contact page">
                  Contact
                </a>
              </li>

            
        
            
              <li class="list f5 f4-ns fw4 dib pr3">
                <a class="hover-white no-underline white-90" href="/services/" title="Services page">
                  Services
                </a>
              </li>

            
        
      </ul>
      



      















    </div>
  </div>
</nav>

    </div>
  </header>


    <main class="pb7" role="main">
      
  <div class="flex-l mt2 mw8 center">
    <article class="center cf pv5 ph3 ph4-ns mw7">
      <header>
        <p class="f6 b helvetica tracked">
          ANALYSES
        </p>
        <h1 class="f1">
          Denovo RNASeq
        </h1>
      </header>
      <div class="nested-copy-line-height lh-copy f4 nested-links nested-img mid-gray">
        <p>We offer the following approach to analyse RNASeq data if the reference genome is not available at public resources like NCBI, ENSEMBL, JGI or others. This might be due to a variety of reasons including lack of a reference altogether amongst many.</p>
<ol>
<li>
<p>Normalization of Reads: Reads will be normalized using a standalone approach to ease the computational load and to correct for coverage of ultra-high/ultra-low kmers to assemble these transcripts.</p>
</li>
<li>
<p>Transcriptome Assembly: At Ikigai, we prefer assemblies on the following approaches: Trinity, RNA-Bloom and rnaSpades. Assemblies can include all the three approaches depending on the requirements. Assembly evaluation will be done using BUSCO.</p>
</li>
<li>
<p>Non-Redundant Catalogue generation: This is done by a successive reduction step to classify assembled transcripts into a primary transcript and its associated alternative transcript, based on sequence identity with coverage, duplicacy and then clustering. Subsequently, these will be categorized into a &ldquo;gene&rdquo; based on quantification and MCL based clustering.</p>
</li>
<li>
<p>Annotation of Assembly: Annotation will be done by using eggNOG-Mapper to assign the features: GO, KEGG pathways, CAZy, COG, closer Orthogroups and various other metrics defined here: <a href="https://github.com/eggnogdb/eggnog-mapper/wiki/eggNOG-mapper-v2">https://github.com/eggnogdb/eggnog-mapper/wiki/eggNOG-mapper-v2</a></p>
</li>
<li>
<p>ncRNA identification: Classification from a broader to a smaller category based on coding potentials and sequence profile using HMMs or other approaches.</p>
</li>
<li>
<p>Diffential Expression: Identification of Significantly modulated genes.</p>
</li>
<li>
<p>GO and KEGG Pathway analyses: Check the over-representation of Ontologies to see which terms are being significantly impacted. KEGG pathways will also be identified to assess the modulated pathways.</p>
</li>
<li>
<p>SuperTranscript Generation: &ldquo;Genes&rdquo; from Step 3, will be used to generate a supertranscript based on iterative clustering and merging to form a contiguous singular representation of a gene. This is akin to getting a gene region from a genome.</p>
</li>
<li>
<p>SSR identification: This is done on the regions encapsulating a gene to identify regions harbouring simple repeat motifs.</p>
</li>
<li>
<p>SNP/InDel and QTL analyses: Variants will be called over the alignments to identify mutations observed under the various experimental conditions and their impact on the genes. These will be used alongwith the Quantification matrix to assess if these variations can be categorized as QTLs.</p>
</li>
</ol>
<p><img src="/uploads/RNASeq_Denovo.svg" alt="RNASeq: Denovo analyses pipeline"></p>

      </div>
    </article>
  </div>

    </main>
    <footer class="bg-black bottom-0 w-100 pa3" role="contentinfo">
  <div class="flex justify-between">
  <a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href="https://ikigaiinformatics.github.io/" >
    &copy;  Ikigai Informatics 2020 
  </a>
    <div>














</div>
  </div>
</footer>

    

  <script src="/dist/js/app.3fc0f988d21662902933.js"></script>


  </body>
</html>
