<!doctype html><html lang=en-us><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge,chrome=1"><title>Reference Aided DNASeq | Ikigai Informatics</title><meta name=viewport content="width=device-width,minimum-scale=1"><meta name=generator content="Hugo 0.75.1"><meta name=ROBOTS content="INDEX, FOLLOW"><link href=https://ikigaiinformatics.github.io/dist/css/app.4fc0b62e4b82c997bb0041217cd6b979.css rel=stylesheet><link rel="shortcut icon" href=https://ikigaiinformatics.github.io/logo.png type=image/x-icon><meta property="og:title" content="Reference Aided DNASeq"><meta property="og:description" content="Variation calling and profiling"><meta property="og:type" content="article"><meta property="og:url" content="https://ikigaiinformatics.github.io/analyses/dnaseq_reference/dnaseq_reference/"><meta itemprop=name content="Reference Aided DNASeq"><meta itemprop=description content="Variation calling and profiling"><meta itemprop=wordCount content="234"><meta itemprop=keywords content><meta name=twitter:card content="summary"><meta name=twitter:title content="Reference Aided DNASeq"><meta name=twitter:description content="Variation calling and profiling"></head><body class="ma0 avenir bg-near-white production"><header><div class=bg-dark-green><nav class="pv3 ph3 ph4-ns" role=navigation><div class="flex-l justify-between items-center center"><a href=https://ikigaiinformatics.github.io/ class="f3 fw2 hover-white no-underline white-90 dib">Ikigai Informatics</a><div class="flex-l items-center"><ul class="pl0 mr3 site-menu"><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=https://ikigaiinformatics.github.io/about/ title="About page">About</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=https://ikigaiinformatics.github.io/bioinformatics/ title="Bioinformatics page">Bioinformatics</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=https://ikigaiinformatics.github.io/it/ title="Computing page">Computing</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=https://ikigaiinformatics.github.io/contact/ title="Contact page">Contact</a></li></ul></div></div></nav></div></header><main class=pb7 role=main><div class="flex-l mt2 mw8 center"><article class="center cf pv5 ph3 ph4-ns mw7"><header><p class="f6 b helvetica tracked">ANALYSES</p><h1 class=f1>Reference Aided DNASeq</h1></header><div class="nested-copy-line-height lh-copy f4 nested-links nested-img mid-gray"><p>We offer the following approach to analyse WGS data if the reference genome is readily available at public resources like NCBI, ENSEMBL, JGI or others.</p><ol><li><p>Alignment of the reference genome: Reads will be aligned to the reference genome to check the mappability, coverage and depth of the organism sequenced.</p></li><li><p>Variation calling: For the samples, we will be performing variant calling to call SNVs, InDels and smaller complex variations like MNPs (&lt;50bp). These will be genotyped to quantify the genetic makeup of the sample.</p></li><li><p>Structural and Copy Number Variation calling: Multiple state of the art tools will be used to extract large-scale variation (>=50bp) events like Duplication, Deletion, Translocation and Inversions.</p></li><li><p>Annotation and Prioritization of Variations: Variations will be annotated to assess their impact on the sequenced sample to check their deleteriousness or viability.</p></li><li><p>Phylogenetic analyses: Concatenated SNPs from each sample or core genes identified by BUSCO will be used for sequence alignments and creating an SNP tree.</p></li><li><p>QTL identificaion: Assuming bulked samples are present, QTLs can be identified from the called variants.</p></li><li><p>GWAS: If many samples (>96) and their associated phenotypes are present, we also perform association testing to see which SNPs can play a major role.</p></li><li><p>Assembly and Annotation of Unaligned reads: Unaliged reads from Step 1 are subjected to assembly to form contiguous regions and are annotated to see if they harbour new genes compared to the reference.</p></li></ol><p><strong>Analyses pipeline at Ikigai</strong>
<link rel=stylesheet href=https://ikigaiinformatics.github.io/css/hugo-easy-gallery.css><div class=box><figure itemprop=associatedMedia itemscope itemtype=http://schema.org/ImageObject><div class=img><img itemprop=thumbnail src=https://ikigaiinformatics.github.io/images/DNASeq_Reference.svg></div><a href=https://ikigaiinformatics.github.io/images/DNASeq_Reference.svg itemprop=contentUrl></a></figure></div></p></div></article></div></main><footer class="bg-dark-green bottom-0 w-100 pa3" role=contentinfo><div class="flex justify-between"><a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href=https://ikigaiinformatics.github.io/>&copy; Ikigai Informatics 2020</a><div></div></div></footer><script src=https://ikigaiinformatics.github.io/dist/js/app.3fc0f988d21662902933.js></script></body></html>