<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>smallRNA Analyses | Ikigai Informatics</title>
    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="generator" content="Hugo 0.75.1" />
    
    
      <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
    

    
    
      <link href="/dist/css/app.4fc0b62e4b82c997bb0041217cd6b979.css" rel="stylesheet">
    

    

    
      

    

    
    
    <meta property="og:title" content="smallRNA Analyses" />
<meta property="og:description" content="We offer the following approach to analyse RNASeq data if the reference genome is readily available at public resources like NCBI, ENSEMBL, JGI or others. We prefer having the genome from ENSEMBL due to the downstream integration with biomart.
  QC and Read Trimming: Trim adapters and retain reads between 18-30bp.
  Alignment to other ncRNAs: ncRNA catalogue from which the smallRNA sequences (precursors or mature) have been removed are used as a bait to remove reads originating from such sequences." />
<meta property="og:type" content="article" />
<meta property="og:url" content="http://ikigaiinformatics.github.io/analyses/details/mirna/" />

<meta itemprop="name" content="smallRNA Analyses">
<meta itemprop="description" content="We offer the following approach to analyse RNASeq data if the reference genome is readily available at public resources like NCBI, ENSEMBL, JGI or others. We prefer having the genome from ENSEMBL due to the downstream integration with biomart.
  QC and Read Trimming: Trim adapters and retain reads between 18-30bp.
  Alignment to other ncRNAs: ncRNA catalogue from which the smallRNA sequences (precursors or mature) have been removed are used as a bait to remove reads originating from such sequences.">

<meta itemprop="wordCount" content="177">



<meta itemprop="keywords" content="" />
<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="smallRNA Analyses"/>
<meta name="twitter:description" content="We offer the following approach to analyse RNASeq data if the reference genome is readily available at public resources like NCBI, ENSEMBL, JGI or others. We prefer having the genome from ENSEMBL due to the downstream integration with biomart.
  QC and Read Trimming: Trim adapters and retain reads between 18-30bp.
  Alignment to other ncRNAs: ncRNA catalogue from which the smallRNA sequences (precursors or mature) have been removed are used as a bait to remove reads originating from such sequences."/>

	
  </head>

  <body class="ma0 avenir bg-near-white">

    

  <header>
    <div class="bg-black">
      <nav class="pv3 ph3 ph4-ns" role="navigation">
  <div class="flex-l justify-between items-center center">
    <a href="http://ikigaiinformatics.github.io/" class="f3 fw2 hover-white no-underline white-90 dib">
      Ikigai Informatics
    </a>
    <div class="flex-l items-center">

      
      
      <ul class="pl0 mr3 site-menu">
        
            
              <li class="list f5 f4-ns fw4 dib pr3">
                <a class="hover-white no-underline white-90" href="/about/" title="About page">
                  About
                </a>
              </li>

            
        
            
              <li class="list f5 f4-ns fw4 dib pr3">
                <a class="hover-white no-underline white-90" href="/contact/" title="Contact page">
                  Contact
                </a>
              </li>

            
        
            
              <li class="list f5 f4-ns fw4 dib pr3">
                <a class="hover-white no-underline white-90" href="/services/" title="Services page">
                  Services
                </a>
              </li>

            
        
      </ul>
      



      















    </div>
  </div>
</nav>

    </div>
  </header>


    <main class="pb7" role="main">
      
  <div class="flex-l mt2 mw8 center">
    <article class="center cf pv5 ph3 ph4-ns mw7">
      <header>
        <p class="f6 b helvetica tracked">
          ANALYSES
        </p>
        <h1 class="f1">
          smallRNA Analyses
        </h1>
      </header>
      <div class="nested-copy-line-height lh-copy f4 nested-links nested-img mid-gray">
        <p>We offer the following approach to analyse RNASeq data if the reference genome is readily available at public resources like NCBI, ENSEMBL, JGI or others. We prefer having the genome from ENSEMBL due to the downstream integration with biomart.</p>
<ol>
<li>
<p>QC and Read Trimming: Trim adapters and retain reads between 18-30bp.</p>
</li>
<li>
<p>Alignment to other ncRNAs: ncRNA catalogue from which the smallRNA sequences (precursors or mature) have been removed are used as a bait to remove reads originating from such sequences. Remaining reads are termed as &ldquo;putative miRNA reads.&rdquo;</p>
</li>
<li>
<p>Alignment of miRNA reads: Reads are aligned to the genome, using a short read aligner and later quantified.</p>
</li>
<li>
<p>Novel miRNA identification: Alignments are also screened and thermodynamically evaluated for the feasibility of secondary structure formation.</p>
</li>
<li>
<p>Diffential Expression: Identification of Significantly modulated genes.</p>
</li>
<li>
<p>Target Prediction: miRNA sequences are used to predict their associated transcript sequences to identify impacted genes.</p>
</li>
<li>
<p>GO and KEGG Pathway analyses of Targetted transcripts: Check the over-representation of Ontologies to see which terms are being significantly impacted. KEGG pathways will also be identified to assess the modulated pathways.</p>
</li>
</ol>
<p><img src="/uploads/smallRNA.svg" alt="smallRNA analyses pipeline"></p>

      </div>
    </article>
  </div>

    </main>
    <footer class="bg-black bottom-0 w-100 pa3" role="contentinfo">
  <div class="flex justify-between">
  <a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href="http://ikigaiinformatics.github.io/" >
    &copy;  Ikigai Informatics 2020 
  </a>
    <div>














</div>
  </div>
</footer>

    

  <script src="/dist/js/app.3fc0f988d21662902933.js"></script>


  </body>
</html>
