<!doctype html><html lang=en-us><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge,chrome=1"><title>Amplicon Analyses | Ikigai Informatics</title><meta name=viewport content="width=device-width,minimum-scale=1"><meta name=generator content="Hugo 0.75.1"><meta name=ROBOTS content="INDEX, FOLLOW"><link href=https://ikigaiinformatics.github.io/dist/css/app.4fc0b62e4b82c997bb0041217cd6b979.css rel=stylesheet><link rel="shortcut icon" href=https://ikigaiinformatics.github.io/logo.png type=image/x-icon><meta property="og:title" content="Amplicon Analyses"><meta property="og:description" content="Community profiling with Amplicons"><meta property="og:type" content="article"><meta property="og:url" content="https://ikigaiinformatics.github.io/analyses/amplicon/amplicon/"><meta itemprop=name content="Amplicon Analyses"><meta itemprop=description content="Community profiling with Amplicons"><meta itemprop=wordCount content="386"><meta itemprop=keywords content><meta name=twitter:card content="summary"><meta name=twitter:title content="Amplicon Analyses"><meta name=twitter:description content="Community profiling with Amplicons"></head><body class="ma0 avenir bg-near-white production"><header><div class=bg-dark-green><nav class="pv3 ph3 ph4-ns" role=navigation><div class="flex-l justify-between items-center center"><a href=https://ikigaiinformatics.github.io/ class="f3 fw2 hover-white no-underline white-90 dib">Ikigai Informatics</a><div class="flex-l items-center"><ul class="pl0 mr3 site-menu"><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=https://ikigaiinformatics.github.io/about/ title="About page">About</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=https://ikigaiinformatics.github.io/bioinformatics/ title="Bioinformatics page">Bioinformatics</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=https://ikigaiinformatics.github.io/it/ title="Computing page">Computing</a></li><li class="list f5 f4-ns fw4 dib pr3"><a class="hover-white no-underline white-90" href=https://ikigaiinformatics.github.io/contact/ title="Contact page">Contact</a></li></ul></div></div></nav></div></header><main class=pb7 role=main><div class="flex-l mt2 mw8 center"><article class="center cf pv5 ph3 ph4-ns mw7"><header><p class="f6 b helvetica tracked">ANALYSES</p><h1 class=f1>Amplicon Analyses</h1></header><div class="nested-copy-line-height lh-copy f4 nested-links nested-img mid-gray"><p>We offer the following approach to analyse Amplicon data if the data is from either Illumina or PacBio CCS platforms. Amplicons are genetic sequences which can be either complete ITS, 16s or 18s or partial segments forming the Hyper-Variable regions.</p><ol><li><p>QC and Read Trimming: Reads are trimmed to remove adapter or primer segments to retain the Amplicon sequence stretch. High quality reads are then retained post the trimming.</p></li><li><p>Merging of the Reads: This is an Illumina only step, where the reads are merged to form a near-complete sequence of the targetted Hyper-Variable region. This leads to the retention of reads between 390-420bp. In case of PacBio CCS data, this is not required, but instead we retain reads greater than 1600bp as it captures the complete gene.</p></li><li><p>Denoising of the Reads: Error profiles are built by pooling the samples and we tend to correct any artifactual errors still present in the reads.</p></li><li><p>Deduplication of the Reads: Once the errors are removed, we use a Single Base resolution to cluster and collapse the reads generating a representative read. These are called as Amplicon Specific Variants.</p></li><li><p>Chimeric Read Removal: Chimeric sequences are identified and eliminated from the Read-pool. Generally, the number of reads removed should be less than 0.5% of the entire dataset.</p></li><li><p>Taxonomic Assignment: Taxonomy for Bacterial and Archaeal sequences are assigned using GTDB and for fungi using UNITE database, though this will depend on the marker gene sequenced.</p></li><li><p>ASV Phylogeny: ASVs passing through the Chimeric Read Removal steps are then aligned using a Multiple-Sequence-Aligner and undergo a Phylogenetic Prediction to determine the ecology covered in the samples.</p></li><li><p>Diversity Analyses: Alpha diversities like Richness, Dominance, Rarefaction, Core-Community are captured and Beta diversities like Community-disparity and Ordination are mined to tease the difference between the samples based on the ASVs quantified and taxonomic assignments.</p></li><li><p>Differential Abundance: Tease out the differences between the various communities and samplings undertaken statistically. These can answer questions regarding impact of drugs on the microbiota, wash of industrial effluents etc.</p></li><li><p>Functional Profiling: Based on the ASVs understand what functional traits (eg KEGG, GO, COG, CAZy etc) are enriched for the communities.</p></li><li><p>Ecological Interaction: Infer co-occurence networks of the various ASVs and decipher patterns of associations between microbial taxa. Identify the key players or Hub microbes that play the most important role.</p></li></ol><p><strong>Analyses pipeline at Ikigai</strong>
<link rel=stylesheet href=https://ikigaiinformatics.github.io/css/hugo-easy-gallery.css><div class=box><figure itemprop=associatedMedia itemscope itemtype=http://schema.org/ImageObject><div class=img><img itemprop=thumbnail src=https://ikigaiinformatics.github.io/images/Amplicon.svg></div><a href=https://ikigaiinformatics.github.io/images/Amplicon.svg itemprop=contentUrl></a></figure></div></p></div></article></div></main><footer class="bg-dark-green bottom-0 w-100 pa3" role=contentinfo><div class="flex justify-between"><a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href=https://ikigaiinformatics.github.io/>&copy; Ikigai Informatics 2020</a><div></div></div></footer><script src=https://ikigaiinformatics.github.io/dist/js/app.3fc0f988d21662902933.js></script></body></html>